pep-0001.txt

```diff
@@ -655,20 +656,23 @@ submit a competing PEP.
 
 If you are interested in assuming ownership of a PEP, you can also do this via
 pull request.  Fork the `PEP repository`_, make your ownership modification,
-and submit a pull request.  You should also send a message asking to take
-over, addressed to both the original author and the PEP editors
-<peps@python.org>.  If the original author doesn't respond to email in a
-timely manner, the PEP editors will make a unilateral decision (it's not like
-such decisions can't be reversed :).
+and submit a pull request.  You should mention both the original author and
+``@python/pep-editors`` in a comment on the pull request.  (If the original
+author's GitHub username is unknown, use e-mail.)  If the original author
```

Comment:
nit: the rest of the document uses "email"
```suggestion
author's GitHub username is unknown, use email.)  If the original author
```

Reply:
Thanks!